gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadPlugin.groovy

```diff
@@ -57,29 +57,30 @@ class SymbolUploadPlugin implements Plugin<Project> {
      * @param target the project to find/create the configuration in
      * @return the project's configuration container
      */
-    private SymbolUploadConfiguration getOrCreateConfig(Project target) {
-        SymbolUploadConfiguration config = target.extensions.findByType(SymbolUploadConfiguration.class)
-        if (config == null) {
-            config = target.extensions.create(CONFIGURATION_KEY, SymbolUploadConfiguration.class)
+    private static SymbolUploadConfiguration getOrCreateConfig(Project target) {
+        SymbolUploadConfiguration config = target.extensions.findByType SymbolUploadConfiguration
+        if (!config) {
+            config = target.extensions.create CONFIGURATION_KEY, SymbolUploadConfiguration
         }
-        return config
+
+        config
```

Comment:
I want my `return` statements too. `config` is a valid expression even when it's not the last line of the function. Without the explicit `return` the compiler won't catch it if someone adds code below this line, which will cause the return value of this function can change unexpectedly.